modules/installation-configure-proxy.adoc

```diff
@@ -237,6 +237,16 @@ bundle.
 The installation program does not support the proxy `readinessEndpoints` field.
 ====
 
+[NOTE]
```

Comment:
In the preview this note doesn't look indented properly. You might need a + on line 239

Reply:
Ack.